docs/src/tool/index.md

```diff
@@ -3,4 +3,363 @@
 This document describes the `reach` tool: how to install it and how to use it.
 
 + @{seclink("ref-install")}
-+ @{seclink("ref-usage")}
\ No newline at end of file
++ @{seclink("ref-usage")}
+
+# {#ref-install} Installation
+
+Reach is designed to work on POSIX systems with [make](https://en.wikipedia.org/wiki/Make_(software)), [Docker](https://www.docker.com/get-started), and [Docker Compose](https://docs.docker.com/compose/install/) installed.
+The best way to install Docker on Mac and Windows is with [Docker Desktop](https://www.docker.com/products/docker-desktop).
+
+:::note
+You probably already have `make` installed.
+For example, OS X and many other POSIX systems come with `make`, but some versions of Linux do not include it by default and will require you to install it.
+If you're on Ubuntu, you can run `sudo apt install make` to get it.
+:::
+
+You can install Reach by running:
+
+```cmd
+$ curl https://docs.reach.sh/reach -o reach ; chmod +x reach
+```
+
+in your project repository.
+You can copy this file to other repositories or move it to a directory in your `PATH`, like `~/bin`.
+(`PATH` is a UNIX environment variable listing each of the directories that contain programs you can run in a shell session.)
+
+:::note
+If you're using Windows, consult [the guide to using Reach on Windows](##guide-windows).
+:::
+
+# {#ref-usage} Usage
```

Comment:
I feel like the name "Usage" is bad now, especially, because this is actually about the version. I think this should be something like
```
# {#ref-usage} Selecting a version
```
or something like that

Reply:
There might come a time when we want to reuse the reference link "usage", I think now would be a good time to modify the ref as well, if you don't object.

Reply:
The name `#ref-usage` has to stay because it is used to generate backwards compatible links with the old site